src/gretel_trainer/relational/report/report_template.html

```diff
@@ -3,91 +3,148 @@
-      table. The individual SQS does not take into account statistical correlations of data across related tables. The
-      cross-table Synthetic Report evaluates the statistical accuracy of the synthetic data of a table with consideration to the
-      correlations between data across related tables. The cross-table SQS provides insight into the accuracy of the
-      table in the context of the database as a whole. More information about the Gretel Synthetic Report and Synthetic Data Quality Score is
+      table. The individual SQS does not take into account statistical correlations of data across 
+      related tables. The
+      cross-table Synthetic Report evaluates the statistical accuracy of the synthetic data of a 
+      table with consideration to the
+      correlations between data across related tables. The cross-table SQS provides insight into 
+      the accuracy of the
+      table in the context of the database as a whole. More information about the Gretel Synthetic 
+      Report and Synthetic Data Quality Score is
       available <a href="https://docs.gretel.ai/reference/synthetics/gretel-synthetic-report">here</a>.</p>
 
-      <h2>Synthetic Data Quality Scores</h2>
+      <span class="h3">Synthetic Data Quality Scores</span>
 
         <p>For each table, individual and cross-table synthetic data quality scores (SQS) are computed and displayed below.</p>
 
         <table>
-          <thead>
-            <tr>
-              <th scope="row">Table</th>
-              <th scope="col">Individual SQS</th>
-              <th scope="col">Cross-Table SQS</th>
-            </tr>
-          </thead>
+          <tr>
+            <th scope="row">Table Name</th>
+            <th scope="col">Individual SQS</th>
+            <th scope="col">Cross-table SQS</th>
+          </tr>
           {% for table, evaluation in presenter.evaluations.items() %}
-          <tbody>
-            <tr>
-              <th scope="row">{{table}}</th>
-              <td style="text-align: center">{{evaluation.individual_sqs}}</td>
-              <td style="text-align: center">{{evaluation.cross_table_sqs}}</td>
-            </tr>
-          </tbody>
+          <tr>
+            <td><b>{{ table }}</b></td>
+            <td>
+              <span class="sqs-table-score">{{ evaluation.individual_sqs }}</span>
+              <span class="label label__{{ presenter.sqs_score_to_grade(evaluation.individual_sqs).lower().replace(' ', '-') }}">
+                {{ presenter.sqs_score_to_grade(evaluation.individual_sqs) }}
+              </span>
+              <span class="sqs-table-link">
+                <a href="synthetics_individual_evaluation_{{ table }}.html">Report</a>
+              </span>
+            </td>
+            <td>
+              <span class="sqs-table-score">{{ evaluation.cross_table_sqs }}</span>
+              <span class="label label__{{ presenter.sqs_score_to_grade(evaluation.cross_table_sqs).lower().replace(' ', '-') }}">
```

Comment:
Could these two (top and bottom of highlighted lines) use the label-maker method as well? (I guess the helper method would have to get reworked a bit since it's calling the same formatting stuff on different properties.)